jerry-core/parser/js/js-parser-scanner.c

```diff
@@ -76,6 +76,9 @@ typedef enum
 #ifndef CONFIG_DISABLE_ES2015_CLASS
   SCAN_STACK_CLASS,                        /**< class language element */
 #endif /* !CONFIG_DISABLE_ES2015_CLASS */
+#ifndef CONFIG_DISABLE_ES2015_FUNCTION_PARAMETER_INITIALIZER
+  SCAN_STACK_FUNCTION_PARAMETERS,          /**< function parameter initializer */
+#endif /* CONFIG_DISABLE_ES2015_FUNCTION_PARAMETER_INITIALIZER */
```

Comment:
Closing comments of `#ifndef`s should be perfixed with `!` (all over the commit).